mantle/cmd/kola/options.go

```diff
@@ -335,6 +339,18 @@ func syncOptions() error {
 func syncCosaOptions() error {
 	switch kolaPlatform {
 	case "qemu-unpriv", "qemu":
+		if kola.QEMUOptions.SecureExecution {
+			// SE needs at least 4GB
+			if kola.QEMUOptions.Memory == "" || kola.QEMUOptions.Memory < "4096" {
```

Comment:
That string comparison won't do the right thing. But I don't think we need it if we implement the next comment.